### PR TITLE
Ecommerce page implementation

### PR DESCRIPTION
**Pull request by asapconet**
Ecommerce page implementation for `Use cases`

### CODE REVIEW

File: components/GetstartedCard/Cards.tsx [hunk 1]
```diff
@@ -44,15 +45,15 @@ const LinkedCard = ({ icon, desc, title, route, }: MyProps) => {
...
-export const PlainCard = ({ desc, title, route, myIcon }: MyProps) => {
+export const PlainCard = ({ desc, title, route, myIcon, className }: MyProps) => {
   return (
+    <>
     <div key={route}>
       <LzLink className='my-2' to={route}>
         <div
           key={route}
-          className='z-10 px-5 mb-2 bg-white md:pt-4 drop-shadow-xs rounded-8 lg:rounded-8 sm:w-auto lg:w-92'
+          className={`z-10 px-5 mb-2 bg-white md:pt-4 drop-shadow-xs rounded-8 lg:rounded-8 sm:w-auto lg:w-92 ${className}`}
```
Comment (Frontend-io) [inline components/GetstartedCard/Cards.tsx:56]: Please use `className()` function here

File: components/GetstartedCard/Cards.tsx [hunk 2]
```diff
@@ -44,15 +45,15 @@ const LinkedCard = ({ icon, desc, title, route, }: MyProps) => {
...
         >
           {myIcon ? myIcon : ''}
```
Comment (Frontend-io) [inline components/GetstartedCard/Cards.tsx:58]:
```js
myIcon ?? ''
```

File: components/mdx-components.tsx [hunk 3]
```diff
@@ -14,10 +14,10 @@ import WhatsNew from './WhatsNew'
 import ConnectCommunity from './Connect'
 import { Pagination } from './pagination'
 import LzAlert from './UI/alert'
-import LinkedCard from './GetstartedCard/Cards'
+import LinkedCard, { PlainCard } from './GetstartedCard/Cards'
 import TransferCard from './TransferCard'
 import BankPayouts from './BankPayouts'
-import { props } from 'ramda'
+import { ShopifyIcon, WooCommerceIcon, WordPressIcon } from 'public/icons'
```
Comment (Frontend-io) [inline components/mdx-components.tsx:20]: Let's not import these directly. Instead, create a component that has the cards and its detail and import the component here instead.

File: content/docs/use-cases/ecommerce.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Dolor pulvinar tincidunt in sed rhoncus lorem. Id eget rhoncus, ipsum mauris nam velit lectus risus. Venenatis non.
+
+<Divider className='w-20'/>
+
+## Ecommerce Platforms
+
+Sed magna nulla pellentesque semper enim.
+
+<div className='grid md:grid-cols-2 gap-8 '>
```
Comment (Frontend-io) [inline content/docs/use-cases/ecommerce.mdx:16]: Please move this inside the component i mentioned above. The mdx file should contain purely Markdown or components, not jsx.

**Review by Frontend-io — CHANGES_REQUESTED** [review]

File: components/GetstartedCard/Cards.tsx [hunk 5]
```diff
@@ -2,13 +2,15 @@
...
 import ArrowRightIcon from 'public/icons/arrow-right-icon'
+import classNames from 'classnames'
 
 interface MyProps {
   desc?: string
   title?: string
   route?: string
   icon?: boolean
-  myIcon?: Object
+  customIcom?: Object
```
Comment (MeisterJustice) [inline components/GetstartedCard/Cards.tsx:12]: a little typo here. `customIcon not customIcom`

**Review by MeisterJustice — CHANGES_REQUESTED** [review]

**Review by MeisterJustice — APPROVED** [review]

**Review by Frontend-io — APPROVED** [review]

**Pull request merged**